README.md

```diff
@@ -1,364 +1,395 @@
+- Putting your Web API in a Docker Container
+- Running the container locally
+- Publishing the container to Docker Hub
+- Publishing the container to ACR (Azure Container Registry)
+- Deploy the container using a web app in azure
+- Setting up Continuous Deployment for your container in a Web App
+- Running the container in Azure using ACI (Azure Container Service)
+- Running the container as part of a cluster using AKS (Azure Kubernetes Service)
 
+We will be pulling together a mixture of hands on labs from the docs and HOLs in this document that will walk you through these steps with commentary.
 
 ## Prerequisites
 
-The first thing you will need to do is to get your machine prepped and ready. 
+The first thing you will need to do is to get your machine prepped and ready.
 
-You will need the following things installed on your system to walk through these exercises.
+You will need the following things installed on your system to walk through these exercises:
 
-Azure CLI [Azure CLI Directions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- Docker for [Windows](https://docs.docker.com/v17.09/docker-for-windows/install/) or [Mac](https://docs.docker.com/v17.09/docker-for-mac/install/)
+- [Docker Hub Account](https://hub.docker.com/) - This is usually created when you install Docker for Mac or Windows and you sign in to the app.
+- Kubectl [Full Directions](https://kubernetes.io/docs/tasks/tools/install-kubectl/). -- [Easiest way](https://docs.microsoft.com/en-us/cli/azure/acs/kubernetes?view=azure-cli-latest) Must have Azure CLI installed
+- [Postman](https://www.getpostman.com/) for testing and making calls to our API
+- If you are running on Mac or want to use VS Code:
+  - [VSCode for Windows, Mac, or Linux](https://code.visualstudio.com/download) or other text editor (Vim, Sublime, etc...)
+  - [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+- [.NET Core 6.0 SDK](https://www.microsoft.com/net/download/all) for running our ASP.NET Core projects
 
-Docker for [Windows](https://docs.docker.com/v17.09/docker-for-windows/install/) or [Mac](https://docs.docker.com/v17.09/docker-for-mac/install/)
+## Creating an ASP.NET Core Web API
 
-[A Dockerhub Account](https://hub.docker.com/) - This is usually created when you install Docker for Mac or Windows and you sign in to the app. 
+The first thing we need to to set up an ASP.NET Core Web API to have something to call when we testing out containers. For our purposes, this needs to be ASP.NET Core since we will be using kubernetes and running Linux containers. If you need to run Windows containers and classic ASP.NET then you could use Azure Service Fabric (not covered in this training) [Service Fabric Quick Start](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-quickstart-containers)
```

Comment:
repeated to..... The first thing we need to to... should be to do

Also this :    when we testing out containers...... should be....    when we are testing our